model/report_sources/proof_of_concept_2020-03-22.Rmd

```diff
@@ -162,7 +171,7 @@ This section contain information on the various parameters. We use these
 data to generate distribution, with discretisation when needed.

```

Comment:
In model description, not obvious what reporting is. Maybe in parameters tab could add all parameters from the user input with a description?
1. augment the number of admissions using the reporting; currently this is just 
$n_{aug} = n_{reported} / reporting$

Also on the html file load packages is empty - could list the packages required?